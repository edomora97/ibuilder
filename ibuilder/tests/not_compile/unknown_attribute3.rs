#[derive(ibuilder::ibuilder)]
struct Foo {
    #[ibuilder(wibble(monster))]
    field: i64,
}

#[derive(ibuilder::ibuilder)]
#[ibuilder(wibble(monster))]
struct Bar {
    field: i64,
}

#[derive(ibuilder::ibuilder)]
#[ibuilder(wibble(monster))]
enum Baz {
    Var,
}

#[derive(ibuilder::ibuilder)]
enum Bim {
    #[ibuilder(wibble(monster))]
    Var,
}

fn main() {}
