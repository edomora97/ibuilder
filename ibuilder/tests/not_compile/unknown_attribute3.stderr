error: unknown attribute
 --> $DIR/unknown_attribute3.rs:3:16
  |
3 |     #[ibuilder(wibble(monster))]
  |                ^^^^^^^^^^^^^^^

error: unknown attribute
 --> $DIR/unknown_attribute3.rs:8:12
  |
8 | #[ibuilder(wibble(monster))]
  |            ^^^^^^^^^^^^^^^

error: unknown attribute
  --> $DIR/unknown_attribute3.rs:14:12
   |
14 | #[ibuilder(wibble(monster))]
   |            ^^^^^^^^^^^^^^^

error: unknown attribute
  --> $DIR/unknown_attribute3.rs:21:16
   |
21 |     #[ibuilder(wibble(monster))]
   |                ^^^^^^^^^^^^^^^
